.circleci/config.yml

```diff
@@ -262,24 +262,80 @@ jobs:
           name: Build production code
           command: make build
           working_directory: ~/project/messaging/relayer
+
+  relayer-test:
+    machine:
+      image: ubuntu-2004:202107-02
+    resource_class: large
+    steps:
+      - checkout
+      - run:
+          name: Verify checkout
+          command: |
+            date -Is
+            echo; git -P reflog --date=iso
+            go version
+            pwd
+            ls -al
+            echo GOROOT: $GOROOT
+      - run:
+          name: Upgrade golang
+          command: |
+            sudo rm -rf /usr/local/go
+            curl -O https://dl.google.com/go/go1.17.1.linux-amd64.tar.gz
```

Comment:
nit: should we use the same go version as the version used by the docker image that runs the tests? i.e. 1.17.3?

Reply:
This will be fixed by addressing the comment below.